src/app/elements/lookup/attachment-modal/attachment-modal.component.html

```diff
@@ -0,0 +1,34 @@
+<div [hidden]="isHidden" class="overlay mt-overlay modal fade" [ngClass]="{'show-modal': !this.isHidden}">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">{{title}}</h4>
+                <button type="button" class="close" aria-label="Close" (click)="hideModal()">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <h6 class="modal-title col-sm-8">{{subTitle}}</h6>
+                <div class="row">
+                    <div clickableImage *ngIf="!isHidden" class="col-sm-8 preview">
+                        <iframe *ngIf="!isImage" [src]="src" frameborder="0"></iframe>
```

Comment:
what will be done if we click on txt?

Reply:
nothing just focus the cursor to the text